### PR TITLE
Modal docs update

### PR DESCRIPTION
**Pull request by 7emansell**
No ticket

## This PR does the following:

- Goal is to improve the Modal Storybook examples and documentation 

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

**Comment by 7emansell** [comment]
@bigfishdesign13 First pass, the section I added is a response to what I always get questions about/mess up in my own code

**Review by EdwinGuzman — APPROVED** [review]

File: src/components/Modal/Modal.mdx [hunk 1]
```diff
@@ -90,7 +91,7 @@ Modal title: Confirm reservation cancellation
 
 Modal description: Your upcoming reservation will be cancelled.
 
-Action buttons: [Cancel] [Confirm]
+Action button: [Confirm]
```
Comment (bigfishdesign13) [inline src/components/Modal/Modal.mdx]: Let's change this to match our newly suggested patterns.
```
Action buttons: [Cancel] [Confirm]
```

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
Right now, is see one small change that is needed.  But what about the concerns that Vera had when she was trying to implement the updated `Modal` component?  Do we need to make additional changes based on her experience?  Do we need to be more explicit about how the props should be used/combined?  Should the required props for each variant be made clearer?

**Review by bigfishdesign13 — APPROVED** [review]
Looks good!

**Pull request merged**